plugins/PostProcessingPlugin/scripts/ColorMix.py

```diff
@@ -60,9 +76,9 @@ def getSettingDataString(self):
                     "default_value": 0,
                     "minimum_value": "0",
                     "minimum_value_warning": "0.1",
```

Comment:
```suggestion
                    "minimum_value_warning": "start_height",
```

Reply:
This will present a warning if the finish_height is smaller than the start_height.

Reply:
good idea